storagemarket/impl/provider_environments.go

```diff
@@ -107,36 +108,8 @@ func (p *providerDealEnvironment) GeneratePieceCommitment(proposalCid cid.Cid, c
 		}
 	}()
 
-	// dump the CARv1 payload of the CARv2 file to the Commp Writer and get back the CommP.
```

Comment:
Moved this code to its own file so it can be used from the test - no logical changes